debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/util/NamingStrategyBase.java

```diff
@@ -0,0 +1,52 @@
+package io.debezium.connector.jdbc.util;
+
+import java.util.Map;
+
+/**
+ * Base class for naming strategies that provides common functionality for applying prefixes, suffixes,
+ * and naming styles (e.g., snake_case, camelCase, etc.) to names.
+ * This class can be extended by naming strategies for columns, collections, or other entities that require
+ * customizable naming conventions.
+ *
+ * @author Gustavo Lira
+ */
+public abstract class NamingStrategyBase {
```

Comment:
I'm not a fan of `Base`, though we have a few instances of its usage across the code base. I prefer using `Abstract` or `AbstractCommon` as a naming prefix. Still, I understand the desire to use suffixes to logically group classes in the IDE.  YMMV.